src/backend/functions/change-requests.ts

```diff
@@ -4,70 +4,94 @@
  */
 
 import { Handler } from '@netlify/functions';
-import { PrismaClient } from '@prisma/client';
+import { PrismaClient, Prisma, CR_Type } from '@prisma/client';
 import {
   ApiRoute,
   ApiRouteFunction,
   API_URL,
   apiRoutes,
   routeMatcher,
+  ChangeRequest,
   buildSuccessResponse,
   buildNotFoundResponse,
-  buildServerFailureResponse
+  buildServerFailureResponse,
+  ChangeRequestType
 } from 'utils';
 
 const prisma = new PrismaClient();
 
+const relationArgs = Prisma.validator<Prisma.Change_RequestArgs>()({
+  include: {
+    submitter: true,
+    wbsElement: true,
+    changes: {
+      include: {
+        implementor: true
+      }
+    },
+    scopeChangeRequest: true,
+    stageGateChangeRequest: true,
+    activationChangeRequest: true
+  }
+});
+
+const convertChangeRequestType = (type: CR_Type): ChangeRequestType =>
+  ({
+    ISSUE: ChangeRequestType.DesignIssue,
+    DEFINITION_CHANGE: ChangeRequestType.NewFunction,
+    OTHER: ChangeRequestType.Other,
+    STAGE_GATE: ChangeRequestType.StageGate,
+    ACTIVATION: ChangeRequestType.Activation
+  }[type]);
+
+const changeRequestTransformer = (
+  changeRequest: Prisma.Change_RequestGetPayload<typeof relationArgs>
+): ChangeRequest => {
+  const wbsNum = {
+    car: changeRequest.wbsElement.carNumber,
+    project: changeRequest.wbsElement.projectNumber,
+    workPackage: changeRequest.wbsElement.workPackageNumber
+  };
+  return {
+    id: changeRequest.crId,
+    submitter: changeRequest.submitter,
+    dateSubmitted: changeRequest.dateSubmitted,
```

Comment:
These fields are the fields you want to avoid having to write out every single one by simply using the spread operator '...' to dump all fields from the changeRequest object into the object you're constructing.